style.scss

```diff
@@ -0,0 +1,516 @@
+
+.fill-container {
+    width: 100%;
+}
+
+blockquote {
+  border-left: 10px solid #9D0E1F;
+  font-style: italic;
+  margin: 1.5em 10px;
+  padding: 0.5em 10px;
+}
+
+.big-link {
+    font-family: 'Share Tech Mono', monospace;
+    font-size: 24px;
+    display: block;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+}
+
+p, li, table {
+    font-family: 'Montserrat', sans-serif;
+}
+
+h1{
+    margin: 0;
+    padding: 25px 0;
+    font-size: 40px;
+    font-family: 'Share Tech Mono', sans-serif;
+    text-align: center;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+
+    /* Adds a hyphen where the word breaks, if supported (No Blink) */
+    -ms-hyphens: auto;
+    -moz-hyphens: auto;
+    -webkit-hyphens: auto;
+    hyphens: auto;
+}
+
+h2, h3{
+    font-family: 'Share Tech Mono', sans-serif;
+    padding: 5px 0;
+}
+
+h4{
+    font-family: 'Share Tech Mono', sans-serif;
+    padding:2px 0;
+}
+
+header h1 {
+    color: white;
+}
+
+html, body{
+    /*=====removes visible scroll bar======*/
+    overflow-x: hidden;
+    overflow-y: hidden;
+    width:100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+section {
+    padding-top: 30px;
+    padding-bottom: 30px;
+    width: 70%;
+    z-index: 1;
+    float: left;
+    position: absolute;
+    height:  100%;
+    max-width:35%;
+    /*padding: 100px 0;*/
+    background: #c9dbe9;
+    background: -webkit-linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    background: -linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    background: -moz-linear-gradient(top, #c9dbe9 0%, #fff 100%);
+    border-top-right-radius: 30% 50%;
+    border-bottom-right-radius: 30% 50%;
+}
+
+/*=======Navigation Within Rainbow Boxes======*/
+.rightNav{
+    float: right;
+    height: auto;
+    width: 100%;
+    z-index:999999;
+    top: 0;
+    left: 0;
+    background-color: #111;
+    overflow-x: hidden;
+	transition-duration: 300ms;
+}
+
+.container{
+    height: 100vh;
+	/*allow boxes to autoresize with page height*/
+	display: flex;
+	flex-direction: column;
+}
+
+.box {
+    flex: 1;
+    text-align: right;
+    justify-content: center;
+}
+
+.box p {
+    font-family: 'Share Tech Mono', monospace;
+    font-size: 4.5vh;
+    color: white;
+    padding-right: 25px;
+}
+
+/*=================Tabcontent generic style====================*/
+
+#tablinkcontainer {
+    float:right;
+    width:100%;
+    transition-duration: 300ms;
+}
+
+.tabcontent {
+	height: 100%;
+    width: 0;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+	overflow-x: hidden;
+    transition: 0.3s;
+}
+
+.tabcontent::-webkit-scrollbar {
+    display: none;
+}
+
+/* ============= Start of SASS for card-specific styling ===========*/
+
+$background-colors: #2ecc71 #f1c40f #e67e22 #ce1126 #9b59b6 #3498db #1abc9c;
+$link-colors: #9b59b6 #3498db #1abc9c #9D0E1F #2ecc71 #f1c40f #e67e22;
```

Comment:
I think that the `#9D0E1F` color is a bit too dark on the red background. The other accent colors work great though

Reply:
i like the color for the closebtn element (the `X`) and for the blockquote bar, but it is way too dark for text links. will play with color schemes later today and fix.